### PR TITLE
feat(FTDA): Add create task api inside rtk query api

### PR DESCRIPTION
**Pull request by faysal-alam-sazim**
### Description of Change
- Added a mutation to create a task
- Added POST api with defined data type
### Associated Ticket Link:
- https://trello.com/c/V0cGOJrR

### CODE REVIEW

File: shared/redux/rtk-apis/tasksAPI.types.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { EPriority, ETaskStatus } from "@/shared/typedefs/enums";
+
+export type TCreateTaskPayload = {
```
Comment (ishmam-iqbal-sazim) [inline shared/redux/rtk-apis/tasksAPI.types.ts:3]:
```suggestion
export type TCreateTaskDto = {
```

File: shared/redux/rtk-apis/tasksAPI.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
 
 import { TTask } from "@/shared/typedefs/types";
+import { TCreateTaskPayload } from "./tasksAPI.types";
```
Comment (ishmam-iqbal-sazim) [inline shared/redux/rtk-apis/tasksAPI.ts:4]: @ and relative imports should be separated by a line.

**Review by ishmam-iqbal-sazim — APPROVED** [review]
Approved pending comments

**Review by ragib-hossain-sazim — APPROVED** [review]
Approved pending comments

**Review by ragib-hossain-sazim — APPROVED** [review]

**Pull request merged**